README.md

```diff
@@ -11,13 +11,13 @@
 
 ## Morpho's vaults
 
-Morpho's vaults represent tokenized supply positions on Morpho. Vaults are compliant to the ERC4626 standard and can be easily integrated. Please refer to the [vaults documentation](https://developers.morpho.xyz/vaults) for more information.
+Morpho's vaults represent tokenized supply positions on Morpho. Vaults are compliant to the ERC4626 standard and can be easily integrated. Please refer to the [vaults documentation](https://developers-vaults.morpho.xyz/) for more information.
```

Comment:
Apparently there is also a little bit of doc concerning the vaults [there](https://docs.morpho.xyz/morphos-ecosystem/user-tutorials/use-vaults-ma-mctoken). Do we want to make it appear ?

Reply:
you are right, will add that as well!